include/mapbox/vector_tile.hpp

```diff
@@ -153,23 +169,19 @@ inline feature::feature(protozero::data_view const& feature_view, layer const& l
     }
 }
 
-inline mapbox::feature::value feature::getValue(const std::string& key) const {
-    auto keyIter = layer_.keysMap.find(key);
-    if (keyIter == layer_.keysMap.end()) {
+inline mapbox::feature::value feature::getValue(const std::string& key, std::string* message ) const {
+    const auto key_count = layer_.keysMap.count(key);
```

Comment:
nit: can we use `layer_.keysMap.equal_range(key)` here?
key_count could be rewritten as:
```
const auto key_range = layer_.keysMap.equal_range(key);
const auto key_count = std::distance(key_range.first, key_range.second);
```